src/snmalloc/backend_helpers/statsrange.h

```diff
@@ -29,36 +28,28 @@ namespace snmalloc
 
       constexpr Type() = default;
 
-      CapPtr<void, ChunkBounds> alloc_range(size_t size)
+      capptr::Arena<void> alloc_range(size_t size)
```

Comment:
That looks like the kind of change that happens to work (because `ChunkBounds` is `ParentRange::ChunkBounds` and we always instantiate this template with a `ParentRange` whose `ChunkBounds` is `Arena`).  Ditto the argument to `dealloc_range` below.

If it is essential that `ParentRange::ChunkBounds` is `Arena`, that can be statically asserted.

Reply:
Sorry, I was moving between main and 0.6.0 as a base, and I brought across a change I shouldn't have.